### ISSUE TITLE
Error combining loom files

### ISSUE THREAD

**Issue by sarawaise** [mojaveazure/loomR#37]
Hello,

Thank you for creating loomR. I'm having difficulty combining two loom files. I'm running R 3.5.2 and the develop version of loomR. All other packages are the most recent version.

I can connect to my files and view the row_attrs/Gene for both with no problems as follows:

```
> lfile <- connect(filename = "file1.loom", mode = "r+")
> lfile[["row_attrs/Gene"]]
Class: H5D
Dataset: /row_attrs/Gene
Filename: <redacted>/file1.loom
Access type: H5F_ACC_RDWR
Attributes: last_modified
Datatype: H5T_STRING {
      STRSIZE 25;
      STRPAD H5T_STR_NULLPAD;
      CSET H5T_CSET_ASCII;
      CTYPE H5T_C_S1;
   }
Space: Type=Simple     Dims=58735     Maxdims=58735
Not chunked

> lfile2 = connect(filename = "file2.loom", mode = "r+")
> lfile2[["row_attrs/Gene"]]
Class: H5D
Dataset: /row_attrs/Gene
Filename: <redacted>/file2.loom
Access type: H5F_ACC_RDWR
Attributes: last_modified
Datatype: H5T_STRING {
      STRSIZE 25;
      STRPAD H5T_STR_NULLPAD;
      CSET H5T_CSET_ASCII;
      CTYPE H5T_C_S1;
   }
Space: Type=Simple     Dims=58735     Maxdims=58735
Not chunked
```

However, I'm unable to combine these two files:

```
> combine(looms = list(lfile, lfile2), filename = "combined.loom", order.by = "Gene")
Validating 2 input loom files
  |==========================================================================| 100%
Error in combine(looms = list(lfile, lfile2), filename = "combined.loom",  : 
  Cannot find 'row_attrs/Gene' in the row attributes for the loom files provided
In addition: Warning message:
In if (!grepl(pattern = order.by, x = row.attrs)) { :
  the condition has length > 1 and only the first element will be used
```

I'm unsure why combine is unable to find 'row_attrs/Gene' for the loom files when I seem to be able to access it ok. Are you able to see what I'm doing wrong?

Best wishes,
Sara

**Comment by mojaveazure** [comment]
Hi Sara,

This is a bug in `combine`, and should be fixed as of commit 1eca16a on the `develop` branch.

**Comment by sarawaise** [comment]
That's great, thank you!
